dags/os_prepare_index.py

```diff
@@ -54,7 +54,7 @@ def arguments(task_id: str) -> List[str]:
     """
 
     with DAG(
-            dag_id=f"es_{env_name}_prepare_index",
+            dag_id=f"os_{env_name}_prepare_index",
```

Comment:
Ça utilise PostgresEnv

Reply:
oui pour lui c'est normal, prepare_index n'interagit pas of opensearch. Mais je peut renommé les vars pour être plus clair.